.github/workflows/ci-push-img.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   push-docker-img:
     name: Create and push docker image
     if: github.repository == 'kubearmor/kubearmor-relay-server'
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Rather than using latest, can we use specific version like 22.04 or 21.04.
The `ubuntu-latest` distribution might change in the future and there are chances our build or worst our runtime env starts failing for unknown reasons.